api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/v3/AddressDTOV3.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.dhsc.htbhf.claimant.model.v3;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.dhsc.htbhf.claimant.model.constraint.NotPattern;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+import static uk.gov.dhsc.htbhf.claimant.regex.PostcodeRegex.CHANNEL_ISLAND_POST_CODE_REGEX;
+import static uk.gov.dhsc.htbhf.claimant.regex.PostcodeRegex.UK_POST_CODE_REGEX_V3;
+
+@Data
+@Builder
+@AllArgsConstructor(onConstructor_ = {@JsonCreator})
+@ApiModel(description = "Multi purpose address object")
+public class AddressDTOV3 {
```

Comment:
Does this differ from the existing addressDTO in the model package?

Reply:
Yes, the postcode regex has changed.